src/extension.ts

```diff
@@ -69,10 +71,10 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
         );
       }
       languageServerUpdater.clear();
-      return clientHandler.stopClients();
+      return clientHandler.stopClient();
```

Comment:
I'm not 100% sure since this is a VS Code command and not a normal function. TBH I don't like stopping/starting the client inside the commands either, and I'm hoping to remove them entirely.